components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIConsumerImpl.java

```diff
@@ -109,9 +109,15 @@ public Subscriber getSubscriber(String subscriberId) throws AppManagementExcepti
         return subscriber;
     }
 
+    /**
+     * Get business owner for a given business owner id.
+     * @param businessOwnerId Id of business owner.
+     * @return
+     * @throws AppManagementException
+     */
     @Override
-    public BusinessOwner  getBusinessOwner(String appId) throws AppManagementException {
-        return appMDAO.getBusinessOwner(appId);
+    public BusinessOwner  getBusinessOwner(int businessOwnerId) throws AppManagementException {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/665ba5bee36f67c4e163ac0422175545ad3a03c5